serve.py

```diff
@@ -126,7 +126,7 @@ async def goto_dandiset_version(request, dataset, version):
         json_info = req.json()
         if json_info is not None:
             id = json_info['_id']
```

Comment:
@mgrauer - will the girder endpoint return the right `_id` here?

alternatively it seems like we don't need to query girder at all and can pass the dataset identifier directly, which would affect both this and the previous route.

Reply:
I agree, we can just pass directly to the gui.